backend/corpora/common/crossref_provider.py

```diff
@@ -0,0 +1,65 @@
+import requests
+from .corpora_config import CorporaConfig
+
+
+class CrossrefProvider(object):
```

Comment:
is there a better package we can put this in to aid discoverability?

Reply:
I was wondering the same. We could create a `providers` package, although I don't see us adding anything else in there?

Reply:
`providers` works for me, even if it's just one class for now.